apps/bixarena/api/src/main/java/org/sagebionetworks/bixarena/api/model/entity/UserEntity.java

```diff
@@ -0,0 +1,96 @@
+package org.sagebionetworks.bixarena.api.model.entity;
+
+import jakarta.persistence.*;
+import java.time.OffsetDateTime;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+@Entity
+@Table(name = "app_user")
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserEntity {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.UUID)
+  private UUID id;
+
+  @Column(unique = true, nullable = false)
+  private String username;
+
+  @Column(unique = true)
+  private String email;
+
+  @Column(name = "first_name")
+  private String firstName;
+
+  @Column(name = "last_name")
+  private String lastName;
+
+  @Enumerated(EnumType.STRING)
+  @Column(nullable = false)
+  @Builder.Default
+  private Role role = Role.user;
+
+  @Column(nullable = false)
+  @Builder.Default
+  private Boolean enabled = true;
+
+  @CreationTimestamp
+  @Column(name = "created_at", nullable = false, updatable = false)
+  private OffsetDateTime createdAt;
+
+  @UpdateTimestamp
+  @Column(name = "updated_at", nullable = false)
+  private OffsetDateTime updatedAt;
+
+  public enum Role {
+    admin,
+    user,
+    readonly,
+    service,
+  }
```

Comment:
The service role will be used when one service sends a request to another (e.g., API → AI service). A read-only role may not be necessary. Role definitions will be implemented in a follow-up PR.